modular-kyma/index.json

```diff
@@ -32,6 +32,6 @@
     "uilayout": "terminal"
   },
   "backend": {
-    "imageid": "kubernetes-kubeadm-2nodes"
+    "imageid": "kubernetes-kubeadm-1node"
```

Comment:
1node has only 2GB and with the lifecycle-manager installing 3 modules, such a small cluster becomes unresponsive very often.